datafusion/core/src/execution/context.rs

```diff
@@ -2296,8 +2235,8 @@ mod tests {
         let url = format!("file://{}", path.display());
 
         let mut table_factories: HashMap<String, Arc<dyn TableProviderFactory>> =
-            HashMap::new();
-        let factory = Arc::new(ListingTableFactory::new(FileType::CSV));
+            RuntimeConfig::default().table_factories;
+        let factory = Arc::new(ListingTableFactory::new());
         table_factories.insert("test".to_string(), factory);
         let rt_cfg = RuntimeConfig::new().with_table_factories(table_factories);
```

Comment:
Probably want an easier way to add/register additional factories (instead of re-providing the entire collection), similar to object_store_registry